.github/workflows/check-pull-request.yml

```diff
@@ -31,12 +31,20 @@ jobs:
           npm run lint
           npm test
 
-      - name: Test Docker Image Build
+      - name: Start docker compose
         run: |
           set +e
-          docker build --no-cache --tag cdp-node-frontend-template .
+          docker compose up --wait-timeout 300 -d --quiet-pull
           exit $?
 
+      - name: Journey tests
+        run: |
+          npm run e2e
```

Comment:
Can we update this prefix to npm run `test:user-journey`?  Or at least `test:e2e`?